infra/update-frontend.sh

```diff
@@ -0,0 +1,17 @@
+# smalls script to test changes to the javascript or html files
```

Comment:
What's the purpose of this script?
(As always, I view adding new bash scripts with some skepticism.... 👀)

Reply:
I don't really love this script but it's super useful for developing the front end. Added a comment- it copies the html and javascript into the output nightly.
Otherwise, you have to run an entire nightly to see changes to the javascript